core/src/main/res/values/arrays.xml

```diff
@@ -105,6 +105,20 @@
         <item>@string/episode_cleanup_never</item>
     </string-array>
 
+    <string-array name="hardware_control_options">
+        <item>@string/hardware_control_fast_forward</item>
+        <item>@string/hardware_control_rewind</item>
+        <item>@string/hardware_control_skip_episode</item>
+        <item>@string/hardware_control_restart_episode</item>
+    </string-array>
+
+    <string-array name="hardware_control_values">
+        <item>FAST_FORWARD</item>
+        <item>REWIND</item>
+        <item>SKIP_EPISODE</item>
+        <item>RESTART_EPISODE</item>
+    </string-array>
```

Comment:
Looking at the PR again, how about simply storing the keycodes here directly? That avoids having to convert them to an enum and back. (sorry for changing my mind)

Reply:
Storing them in a `@integer/` constant could help to make the `android:default` in the settings xml a bit easier to read.